apps/studio/prisma/generated/types.ts

```diff
@@ -0,0 +1,52 @@
+import type { ColumnType } from "kysely"
+export type Generated<T> =
+  T extends ColumnType<infer S, infer I, infer U>
+    ? ColumnType<S, I | undefined, U>
+    : ColumnType<T, T | undefined, T>
+export type Timestamp = ColumnType<Date, Date | string, Date | string>
+
+export type Accounts = {
+  id: string
+  provider: string
+  providerAccountId: string
+  user_id: string
```

Comment:
hmm this means that we will end up using snake case throughout our codebase. Is there a setting we can use to make camelCase instead? (iirc, that is the default, not sure why this is snek case)

Reply:
hmm this represents our DB right, which is in snake_case?

Reply:
https://github.com/valtyr/prisma-kysely
think we just need to enable a flag for the camel case plugin bah

Reply:
i removed the `@map` on the schema level, so all is in camelCamse now